mappings/net/minecraft/util/DyeColor.mapping

```diff
@@ -6,6 +6,7 @@ CLASS net/minecraft/unmapped/C_arllgqae net/minecraft/util/DyeColor
 	FIELD f_nawrgolb name Ljava/lang/String;
 	FIELD f_pifojmig mapColor Lnet/minecraft/unmapped/C_ifcgjcqy;
 	FIELD f_rhmismvc colorComponents [F
+	FIELD f_xhygajpx CODEC Lnet/minecraft/unmapped/C_lgkqzafw$C_nxwenkbc;
```

Comment:
Other CODEC fields were made non-capital in a previous one of your PRs, why not this one?

Reply:
the warnings pr goes after codec fields that are not `static final`, this one is